README.rst

```diff
@@ -149,22 +149,32 @@ Towncrier has the following global options, which can be specified in the toml f
     underlines = "=-~"
     wrap = false  # Wrap text to 79 characters
     all_bullets = true  # make all fragments bullet points
+    fragment_types = [] # A list of strings with custom fragment types. Leave it empty if you want to use the default ones.
```

Comment:
I  think that due to lack the of the docs documentation, we ended up with documentation split between README and docs/

It would be nice to have fewer docs in the README.
README should just be a quick introduction to all the pointers for the project - docs, download, contributing... etc


```suggestion
    fragment_types = [] # A list strings to overwrite the default ones. If you want to reuse the default types you need to include them here. Leave it empty if you want to use the default ones.
```